OA-PASS/pass-docker
# Move Ember to /app

## Overview
* Adds build-time variable `EMBER_ROOT_URL` for baking the root URL in the Ember image
* Configures Ember to be at `/app/` by default
* Configures Apache proxies to proxy/redirect to the right place
* Points to the latest Ember from the tip of `develop`

## To Test
* Do `docker-compose pull` then `docker-compose up`
* Point your browser to `https://pass/app`
* Make sure nothing is broken
